homeassistant/components/calendar/__init__.py

```diff
@@ -518,6 +519,13 @@ class CalendarEntity(Entity):
 
     _alarm_unsubs: list[CALLBACK_TYPE] | None = None
 
+    _attr_color: RGBColor | None = None
+
+    @property
+    def color(self) -> RGBColor | None:
```

Comment:
This is currently untested. I think the `_attr_color` function needs to call `self.color` and not get the attribute directly in case the integration overrides the color property and doesn't use the attribute.

Reply:
In `state_attributes` call `self.color` instead of `self._attr_color`

Reply:
done